.github/workflows/contract.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       - name: Audit
         run: yarn audit --groups dependencies
 
+      - name: Copy and rename the .env.example file

```

Comment:
Is this step needed? This copies the value but doesn't add any env vars. And for GHA the env vars should be coming from the GHA settings right?

Reply:
The values don't matter here, they are just needed so that the hardhat config doesn't throw undefined errors. none of them are used.

Reply:
Maybe we set defaults on the hardhat config? Doesn't sound like a good reason to add this step. Especially because it will confuse the build process for someone that's looking at this.

Reply:
I can do that.